js/main.js

```diff
@@ -0,0 +1,36 @@
+const div = document.createElement('div');
+const grayBgDiv = document.createElement('div');
+const hamburgerBttn = document.querySelector('.menu-bars');
+
+hamburgerBttn.addEventListener('click', () => {
+  div.id = 'overlay';
+  div.classList.add('overlay');
+  grayBgDiv.id = 'overlay-menu-bg';
+  grayBgDiv.classList.add('overlay');
+  div.innerHTML = '<img src="./media/cancel.png" class="cancelBttn"></img>'
+    + '<ul class="mobile-menu-ul">'
+    + '<li class="mobile-menu-li"><a href="#projects">Porfolio</a></li>'
+    + '<li class="mobile-menu-li"><a href="#about-me">About</a></li>'
+    + '<li class="mobile-menu-li"><a href="#contact">Contact</a></li>'
+    + '</ul>';
+  document.body.appendChild(div);
+  document.body.appendChild(grayBgDiv);
+  div.style.display = 'block';
+  grayBgDiv.style.display = 'block';
+
+  // Add listener for closing menu
+  const mobileMenu = document.querySelector('.cancelBttn');
+  mobileMenu.addEventListener('click', () => {
+    div.remove();
+    grayBgDiv.remove();
+  });
+
```

Comment:
Last but not least you have a really great code and functionality here, very good job 😄

Reply:
Thanks, Juan! Appreciate your feedback alot!